frontend/src/components/admin/stopsEditing.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+<div class="container">
+    <div class="columns">
+
+        <div class="column is-8">
+            <div v-if="failure" class="notification is-danger">
+                <p>There was an error saving the route.</p>
+            </div>
+            <div v-if="success" class="notification is-success">
+                <p>Route saved successfully</p>
+            </div>
+
+            <map-view v-if="false && !creation"></map-view>
+            <place-stop v-if="creation"/>
+        </div>
+        <div class="form-horizontal column" >
+            
+            <!-- Text input-->
+            <div class="field">
+            <label class="label" for="Name">Name</label>
+            <div class="control">
+                <input v-model="stop.name" id="Name" name="Name" type="text" placeholder="Name" class="input" :disabled="!creation">
+            </div>
+            </div>
+
+            <!-- Text input-->
+            <div class="field">
+            <label class="label" for="Description">Description</label>
+            <div class="control">
+                <input v-model="stop.description" id="Description" name="Description" type="text" placeholder="Description" class="input " :disabled="!creation">
+            </div>
+            </div>
+
+            <!-- Text input-->
+            <div class="field">
+            <label class="label" for="Latitude">Latitude</label>
+            <div class="control">
+                <input v-model="stop.latitude" id="Latitude" name="Latitude" type="text" placeholder="Latitude" class="input " :disabled="!creation">
```

Comment:
Reactivity doesn't appear to be working on these textboxes

Reply:
Reactive form was not implemented, don't think it's an issue?

Reply:
well the v-model is supposed to make the box reactive, It is confusing to have -1 in the box when that is not the value being posted.

Reply:
Resolved